accurest-core/src/test/groovy/io/codearte/accurest/builder/SpockMethodBuilderSpec.groovy

```diff
@@ -169,4 +169,42 @@ class SpockMethodBuilderSpec extends Specification {
 			blockBuilder.toString().contains("responseBody.property2 ==~ java.util.regex.Pattern.compile('[0-9]{3}')")
 	}
 
+	def "should generate a call with an url path and query parameters"() {
+		given:
+		GroovyDsl contractDsl = GroovyDsl.make {
+			request {
+				method 'GET'
+				urlPath('/users') {
+					queryParameters {
+						parameter 'limit': $(client(equalTo("20")), server(equalTo("10")))
```

Comment:
Looks very good by one more question though - if client says `equalTo(20)` then on the server side shouldn't it be also be `20`?